step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/AutomationPackageManagerOS.java

```diff
@@ -81,8 +81,8 @@ public static AutomationPackageManager createIsolatedAutomationPackageManagerOS(
                 new InMemoryPlanAccessor(),
                 resourceManager,
                 new InMemoryExecutionTaskAccessor(),
-                new AutomationPackageHookRegistry(), reader
-        );
+                new AutomationPackageHookRegistry(), reader,
+                new AutomationPackageLocks(60));
```

Comment:
Introduce a constant like DEFAULT_READLOCK_TIMEOUT_SECONDS instead of using magic numbers.